x-pack/plugins/security/server/authentication/providers/kerberos.ts

```diff
@@ -311,7 +316,7 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
         // means (e.g. when anonymous access is enabled in Elasticsearch).
         authorization: `Negotiate ${Buffer.from('__fake__').toString('base64')}`,
       });
-      this.logger.debug('Request was not supposed to be authenticated, ignoring result.');
+      this.logger.error('Request was not supposed to be authenticated, ignoring result.');
```

Comment:
Sorry for not providing a context. Let me copy the comment from this file:

>   // We should send a fake SPNEGO token to Elasticsearch to make sure Kerberos realm is included
        // into authentication chain and adds a `WWW-Authenticate: Negotiate` header to the error
        // response. Otherwise it may not be even consulted if request can be authenticated by other
        // means (e.g. when anonymous access is enabled in Elasticsearch).

I'm logging an error if Kibana is configured with Kerberos, but for some reason, Kibana cannot reach the Kerberos realm in Elasticsearch while authenticating the request (either it's not configured in Elasticsearch at all or just misconfigured). I'm not sure why I didn't initially log this with an `ERROR` level.